### DISCUSSION TITLE
Is this still in development?

### DISCUSSION THREAD

**Discussion by buildwithtausif** [manucaralmo/GlowCookies#21] [Ideas]
Hi Devs!
I wanna know that if this project is still in development or not? If is in development?
I would recommend you to please do something like, make cookie-scanner functionality which than sets the available cookies automatically in the advanced cookie setting area and so on we know what to do acording to EU-GDPR laws.
Please do this in next update if you're thinking of making it compatible with EU-GDPR laws.

**Comment by manucaralmo** [comment]
Hi! The project is still under development and we are working on a new update with advanced cookie and script management. We will consider your proposal! Thank you!
